husky_gazebo/controller/arm_controller_ur5.yaml

```diff
@@ -0,0 +1,29 @@
+arm_controller:
```

Comment:
All this stuff should be in husky_control repo, since it'll be used for sim and live probably correct?